container-chains/templates/simple/runtime/src/lib.rs

```diff
@@ -432,8 +445,43 @@ impl Contains<RuntimeCall> for MaintenanceFilter {
 /// This can change in the future
 pub struct NormalFilter;
 impl Contains<RuntimeCall> for NormalFilter {
-    fn contains(_c: &RuntimeCall) -> bool {
-        true
+    fn contains(c: &RuntimeCall) -> bool {
+        match c {
+            // Only allow force_default_xcm_version
+            RuntimeCall::PolkadotXcm(method) => match method {
```

Comment:
This is actually not something we need, as we dont want to prohibit calls to polkadotXcm